ios/RCTImageResizer/RCTImageResizer.m

```diff
@@ -76,6 +113,11 @@ bool saveImage(NSString * fullPath, UIImage * image, NSString * format, float qu
             }
         }
 
+        // Rotate image if rotation is specified.
+        if ((rotation <= -1) || (rotation >= 1)) {
```

Comment:
Is there a particular reason not for having this simpler condition `rotation != 0`? :)